src/Controls/src/Core/TitleBar/TitleBar.cs

```diff
@@ -546,6 +557,40 @@ static View BuildDefaultTemplate()
 			visualStateGroups.Add(trailingContentVisibleGroup);
 			#endregion
 
+			#region FlowDirection states
+			var flowDirectionGroup = new VisualStateGroup() { Name = "FlowDirectionGroup" };
+
+			// Left-to-Right state (default)
+			var ltrState = new VisualState() { Name = TitleBarLTRState };
+			ltrState.Setters.Add(new Setter()
+			{
+				Property = MarginProperty,
+				TargetName = TemplateRootName,
+#if MACCATALYST
+				Value = new Thickness(80, 0, 0, 0)  // System buttons on left in macOS
```

Comment:
Could extract magic numbers to Constants? Also, include a detailed comment explaining the values.

Reply:
@jsuarezruiz, I’ve extracted the numeric values into const variables and added appropriate comments.